dagshub/models/model_locator.py

```diff
@@ -0,0 +1,320 @@
+import logging
+import os
+from dataclasses import dataclass
+from enum import auto, IntEnum
+from functools import cached_property
+from pathlib import Path, PurePosixPath
+from typing import Optional, Literal, Union, Tuple, List
+
+import dacite
+import yaml
+from pathvalidate import sanitize_filepath
+
+from dagshub.common.analytics import send_analytics_event
+from dagshub.common.api import RepoAPI
+from dagshub.common.api.repo import PathNotFoundError
+from dagshub.common.api.responses import StorageAPIEntry
+from dagshub.common.determine_repo import determine_repo
+from dagshub.common.helpers import log_message
+from dagshub.models.model_loaders import ModelLoader, RepoModelLoader, BucketModelLoader, DagsHubStorageModelLoader
+
+# TODO:
+"""
+Ways to get model:
+- automatic - with the cascade
+- from path
+- from bucket
+- from mlflow model
+- from mlflow artifact
+"""
```

Comment:
What is left to do? 
Is it part of the PR?

Reply:
forgot to delete it.
Left to do: mlflow